docs/api/07_coupons/update-coupon.mdx

```diff
@@ -117,19 +120,25 @@ import TabItem from '@theme/TabItem';
     "amount_cents": 5000,
     "amount_currency": "USD",
     "expiration": "time_limit",
-    "expiration_duration": 10
+    "expiration_date": "2022-08-08",
+    "coupon_type": "fixed_amount",
+    "frequency": "recurring",
+    "frequency_duration": 3
   }
 }
 ```
 
 | Attributes | Type | Description |
 |--|--|--|
-| name | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Coupon name |
-| code | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Code identifying the coupon<br></br>It won't be updated if coupon is attached to any subscriptions |
-| amount_cents | Integer &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Amount of the coupon<br></br>It won't be updated if coupon is attached to any subscriptions |
-| amount_currency | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Currency of the amount<br></br>It won't be updated if coupon is attached to any subscriptions |
-| expiration | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Expiration type.<br></br>It can be `time_limit` or `no_expiration`<br></br>It won't be updated if coupon is attached to any subscriptions |
-| expiration_duration | Integer &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Coupon duration<br></br>It won't be updated if coupon is attached to any subscriptions |
+| name | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Coupon name |
+| code | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Code identifying the coupon |
+| amount_cents | Integer &nbsp &nbsp &nbsp<Optional>**Optional (required only for `fixed_amount` type)**</Optional> | Amount of the coupon |
+| amount_currency | String &nbsp &nbsp &nbsp<Optional>**Optional (required only for `fixed_amount` type)**</Optional> | Currency of the amount |
+| expiration | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Expiration type.<br></br>It can be `time_limit` or `no_expiration` |
+| expiration_date | String &nbsp &nbsp &nbsp<Optional>**Optional (required only for `time_limit` type)**</Optional> | Coupon expiration date. |
+| frequency | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Frequency type.<br></br>It can be `once` or `recurring` |
+| frequency_duration | String &nbsp &nbsp &nbsp<Optional>**Optional (required only for `recurring` frequency type)**</Optional> | This field determines number of billing periods to which the coupon is applied. |
```

Comment:
This field determines the number of billing periods to which the coupon applies.